starting/sast.mdx

```diff
@@ -13,28 +13,28 @@ Além do Android Studio, é fundamental instalar ferramentas complementares que
 Cada uma dessas ferramentas tem um papel importante no processo de engenharia reversa, permitindo a análise de como o aplicativo funciona, quais dependências ele utiliza e como os dados são manipulados.
 
 ### JADX
```

Comment:
Use `##` (h2) nos headings para manter a consistência com as outras páginas e talvez seja interessante contextualizar um pouco mais das ferramentas. E responder as seguintes perguntas: 

- Qual a diferença entre JD-GUI e JADX? 
- Devo usar os dois?

Não precisa criar um tópico para responder cada item, conforme você for descrevendo cada ferramenta tente abordar de forma sutil.